### ISSUE TITLE
Provider template has many unused dependencies

### ISSUE THREAD

**Issue by brooksmtownsend** [wasmCloud/project-templates#25]
Here's a list of dependencies I can eliminate and still build the provider.
```
async-trait = "0.1"
base64 = "0.13"
bytes = "1.0"
chrono = "0.4"
crossbeam="0.8"
futures = "0.3"
once_cell = "1.8"
reqwest = "0.11"
rmp-serde = "0.15"
serde_bytes = "0.11"
serde_json = "1.0"
serde = {version = "1.0", features = ["derive"] }
thiserror = "1.0"
tokio = { version = "1", features = ["full"] }
toml = "0.5"
```

These should be removed if they are unused

**Comment by brooksmtownsend** [comment]
cc @stevelr

**Comment by stevelr** [comment]
Fixed in PR #27
